packages/executor/src/config.ts

```diff
@@ -293,6 +294,12 @@ export class Config {
       )
     );
 
+    config.fastlaneValidators = fromEnvVar(
+      "FASTLANE_VALIDATOR",
+      config.fastlaneValidators || bundlerDefaultConfigs.fastlaneValidators,
```

Comment:
This in theory will be used only during the testing phase, no?

Reply:
yes, that's correct